### ISSUE TITLE
Binary Classifier experiment - log loss

### ISSUE THREAD

**Issue by andrewcz** [automl/auto-sklearn#61]
Hi,
Fantastic Library!
I was just wondering, i am trying to use the library for a binary classifier experiment using the log loss function to train the model. This is for a university experiment around benchmarking different models. Would you have time to provide an example of how to use the library to achieve the above goal.
Also, with an included visualisation on how the model learns.
Many thanks,
Best,
Andrew

**Comment by mfeurer** [comment]
Hi,

we include only scikit-learns [SGDClassifier](http://scikit-learn.org/stable/modules/generated/sklearn.linear_model.SGDClassifier.html) as a classifier that can use the log-loss. Nevertheless, there are several other loss functions possible in the SGDClassifier, and it is not possible to restrict their use without touching the source code. You can include logistic regression into the available classifiers for your project, please have a look at the [manual](http://auto-sklearn.readthedocs.org/en/master/extending.html#extending). You have to put the wrapper in the installd auto-sklearn directory where the other classifiers are located, and tell auto-sklearn to only use your method with:

    AutoSklearnClassifier(include_estimators=['custom_logistic_regression']

where custom_logistic_regression is the name of your python module.

auto-sklearn does not provide visualizations, and it is not the scope of the project to provide visualizations for how a single model learns. Please figure out whether scikit-learn provides this functionality.

**Comment by mfeurer** [comment]
See the referencing issue for reasons of closing this issue.
